.github/workflows/build.yaml

```diff
@@ -143,3 +146,47 @@ jobs:
           APICULTURIST_GITHUB: ${{ secrets.APICULTURIST_GITHUB }}
           APICULTURIST_TOKEN: ${{ secrets.APICULTURIST_TOKEN }}
           monorepo: ${{ inputs.monorepo }}
+
+  dev_build:
+    runs-on: ubuntu-latest
+    needs: read_package
+    name: ${{ matrix.repos }}
+    strategy:
+      fail-fast: false
+      matrix:
+        repos: ${{fromJSON(needs.read_package.outputs.repoNames)}}
+    steps:
+      - name: Setup workspace
+        uses: qlik-oss/sn-gh-workflows/actions/setup@v3.0.1
+        with:
+          GH_ACCESS_TOKEN: ${{ secrets.GH_ACCESS_TOKEN }}
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN || secrets.NPM_CI_TOKEN }}
+          NPMRC_ENCODED: ${{ secrets.QLIK_NPM_DEV_ENCODED }}
+      - name: Dev build
```

Comment:
We shouldn't have to build twice. Builds from previous build step should be archived and used here instead

Reply:
We only have a normal build in the previous step (which is the one we want to use for testing I assume)
This is doing a dev build to get the import map included